CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file. See [standa
 
 ## [Unreleased](https://github.com/dotenv-org/python-dotenv-vault/compare/v0.5.1...master)
 
+## 0.7.0
+
+### Changed
+
+- Allow to place .env.vault file anywhere in app [#13](https://github.com/dotenv-org/python-dotenv-vault/pull/13)
```

Comment:
Update message to: 

Look for `.env.vault` file at same location as `.env` file. Finds `.env` file anywhere in app (just like original python lib)

Reply:
changed!